### PR TITLE
Case study: Estimating Dynamic Cross-National Opinion Using Existing Survey Data

### PR DESCRIPTION
**Pull request by cbpclaassen**

### CODE REVIEW

File: users/documentation/case-studies.md [hunk 1]
```diff
@@ -31,6 +31,28 @@ for code, Creative Commons for text)</span>; authors retain all copyright.
...
+class="note">(HTML)</span>
+
+Authors
+: Christopher Claassen
+
+Keywords
+: latent variable models, public opinion, longitudinal model
+
+Source Repository
+: [example-models/education/dynamic-opinion](https://github.com/joonho112/example-models/tree/master/education/dynamic-opinion)
```
Comment (jgabry) [inline users/documentation/case-studies.md:47]: This link gives me a page not found error. Is that the right url?

**Review by jgabry — CHANGES_REQUESTED** [review]
Looks good except for the one issue with the url (see review comment)

File: users/documentation/case-studies.md [hunk 2]
```diff
@@ -31,6 +31,28 @@ for code, Creative Commons for text)</span>; authors retain all copyright.
...
+class="note">(HTML)</span>
+
+Authors
+: Christopher Claassen
+
+Keywords
+: latent variable models, public opinion, longitudinal model
+
+Source Repository
+: [example-models/knitr/dynamic-opinion](https://github.com/joonho112/example-models/tree/master/knitr/dynamic_opinion/estimating-dynamic-latent-opinion.html)
```
Comment (jgabry) [inline users/documentation/case-studies.md:47]: I'm still getting an error even with this new link. It doesn't look like there's a dynamic-opinion folder inside `joonho112/example-models/tree/master/knitr/`.

**Comment by cbpclaassen** [comment]
Hi Jonah,

Sorry, I don't think I did the pull request properly. I've tried again -
hopefully its now available to you

best,
Chris

**Comment by cbpclaassen** [comment]
I've tried once more to push the files to your master repo. Probably did something wrong first time, sorry!

File: users/documentation/case-studies.md (hunk 2 above)
Comment (cbpclaassen) [reply users/documentation/case-studies.md:47]: I've changed the url to my repo - maybe the path of least resistance?

Comment (jgabry) [reply users/documentation/case-studies.md:47]: Sorry for the delay on this. The current URL I see is 

https://github.com/joonho112/example-models/tree/master/knitr/dynamic_opinion/estimating-dynamic-latent-opinion.html

but this still gives me a 404 - page not found error. Is the URL right? Or is this a private repository maybe?

**Comment by jgabry** [comment]
Closing (with author's permission) in favor of #196

**Pull request closed without merging**